bundle-workflow/tests/tests_system/test_arch.py

```diff
@@ -15,6 +15,18 @@ class TestArch(unittest.TestCase):
     def test_current_arch(self):
         self.assertTrue(current_arch() in ["x64", "arm64"])
 
+    @patch("subprocess.check_output")
+    def test_x64_arch(self, mock_subprocess):
+        mock_subprocess.return_value = "x86_64".encode()
+        self.assertTrue(current_arch() == "x64")
+
+    @patch("subprocess.check_output")
+    def test_arm64_arch(self, mock_subprocess):
```

Comment:
Split into 2 tests.

Reply:
I will address this in another improvement PR.